extra-admin/zram-generator/autobuild/beyond

```diff
@@ -0,0 +1,10 @@
+abinfo "Making systemd-service and man ..."
+make systemd-service man
+
+abinfo "Installing systemd-service and man ..."
+make install DESTDIR="$PKGDIR" NOBUILD=1
```

Comment:
What does NOBUILD=1 mean?

Reply:
`NOBUILD=1` here means `make install` will not install `zram-generator` to `/usr/bin`. I think it's necessary because:
- our Rust build script has done it before this step
- in this case, built binary should be placed in `/usr/lib/systemd/systemd-generator` instead
So marking `NOBUILD=1` can reduce redundant and unnecessary actions.